mne/annotations.py

```diff
@@ -880,6 +910,29 @@ def _check_event_id(event_id, raw):
         raise ValueError('Invalid input event_id')
 
 
+def _check_event_description(event_desc, events):
+    """Check event_id and convert to default format."""
+    # check out event_id dict
+    if event_desc is None:  # convert to int to make typing-checks happy
+        event_desc = list(np.unique(events[:, 2]))
+
+    if isinstance(event_desc, np.ndarray):
```

Comment:
you can use:

```
from collections.abc import Iterable
if isinstance(event_desc, dict):
     
elif isinstance(event_desc, Iterable)

elif callable(event_desc):

else:
    raise ...
```

I feel it would be more readable

Reply:
Thing is `Iterable` also includes str, which we can't have here...